vaadin-list-box-flow-parent/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java

```diff
@@ -32,7 +32,7 @@
  *            type of the item represented by this component
  */
 @Tag("vaadin-item")
-@NpmPackage(value = "@vaadin/vaadin-item", version = "2.3.0")
+@NpmPackage(value = "@vaadin/vaadin-item", version = "2.3.0-beta1")
```

Comment:
Is this right?

Reply:
Currently in master https://github.com/vaadin/vaadin-list-box-flow/blob/master/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java#L35